README.md

```diff
@@ -1,193 +1,48 @@
-  * Click the `Launch Instance` button
-  * Provide the required details
-  * Optionally, select the Advanced tab and specify a segment name in the `Romana Network Segment` field
-  * Click the Launch button
-- launch an instance using command-line: `nova boot --flavor m1.nano --image cirros-0.3.4-x86_64-uec --nic net-id=$(neutron net-show romana -Fid -f value) --meta romanaSegment=default instance-name`
-- connect to the instance: `ssh cirros@instance-ip`
-- install an ubuntu image:
-  * Create a new flavor with RAM:512MB, Disk: 3GB, VCPUs: 1: `nova flavor-create m1.smallish auto 512 3 1`
-  * Download a suitable image: `wget https://cloud-images.ubuntu.com/releases/14.04.3/release/ubuntu-14.04-server-cloudimg-amd64-disk1.img`
-  * Create image: `glance image-create --visibility public --disk-format qcow2 --container-format bare --name "ubuntu" < ubuntu-14.04-server-cloudimg-amd64-disk1.img`
-- register a new host: on the controller, run `romana add-host <hostname> <host-ip> <romana-cidr> <agent-port>`
-- add a new tenant: on the controller, run `romana create-tenant <tenant-name>`
-- add a new segment: on the controller, run `romana add-segment <tenant-name> <segment-name>`
-- see a list of tenants: `romana show-tenant`, details for a specific tenant: `romana show-tenant <tenant-name>`
-- see a list of hosts: `romana show-host`, details for a specific host: `romana show-host <hostname>`
+#  Installation
 
-See also: [Try Romana Now](http://romana.io/try_romana/openstack/)
+To get up and running with Romana, some scripts and Ansible playbooks have been provided to automate the setup and deployment.
+It currently supports [Amazon EC2](https://aws.amazon.com/ec2/) instances and [Vagrant](https://www.vagrantup.com/) VMs,
+integrated with either [Kubernetes](http://kubernetes.io) or [Devstack](http://docs.openstack.org/developer/devstack/).
 
-## Working with the code
+* [Romana on AWS EC2 with Kubernetes](aws_kubernetes.md)
+* [Romana on AWS EC2 with Devstack](aws_devstack.md)
+* [Romana on Vagrant VMs with Kubernetes](vagrant_kubernetes.md)
+* [Romana on Vagrant VMs with Devstack](vagrant_devstack.md)
 
-This repository here contains the demo installer and documentation. The actual
-source code, however, is contained in these two repositories:
+Additional installation platforms are being targeted.
+You can express your interest in specific platforms or get help with manually installing Romana by [contacting us](#contact-us).
 
-* [core](https://github.com/romana/core ): A number of micro services written in Go, which comprise the core components of the Romana system.
-* [networking-romana](https://github.com/romana/networking-romana): The Romana ML2 plugin an IPAM driver for OpenStack.
+# Using Romana
 
-The READMEs of those repos contain more information about the source code and
-how to run and test it.
+Now that Romana is installed, choose your stack type below for more information on what you can do.
```

Comment:
How about: "Once you have Romana installed and running in a cluster, you might like to test its capabilities and see it in action. The two links below give you cluster specific suggestions of what to try and what to explore and look at."